apps/browser/src/autofill/content/autofillv2.ts

```diff
@@ -889,42 +396,54 @@ function fill(document: Document, fillScript: AutofillScript) {
      * @argument ops An array of FillScripts to execute
      * @argument theOperation A callback to execute after the operations are complete (this appears to be misnamed)
      */
-    doOperation = function (ops: FillScript[], theOperation) {
-      var op = ops[0];
-      if (void 0 === op) {
+    const doOperation = function (ops: FillScript[], theOperation: () => void): void {
+      let op = ops[0];
+
+      if (op === void 0) {
         theOperation();
       } else {
         // should we delay?
-        if ("delay" === (op as any).operation || "delay" === op[0]) {
+        if ((op as any).operation === "delay" || op[0] === "delay") {
           operationDelayMs = (op as any).parameters ? (op as any).parameters[0] : op[1];
         } else {
           if ((op = normalizeOp(op))) {
-            for (var opIndex = 0; opIndex < op.length; opIndex++) {
-              -1 === operationsToDo.indexOf(op[opIndex]) && operationsToDo.push(op[opIndex]);
+            for (let opIndex = 0; opIndex < op.length; opIndex++) {
+              operationsToDo.indexOf(op[opIndex]) && operationsToDo.push(op[opIndex]) === -1;
             }
           }
+
           theOpIds = theOpIds.concat(
             operationsToDo.map(function (operationToDo) {
+              // eslint-disable-next-line no-prototype-builtins
               return operationToDo && operationToDo.hasOwnProperty("opid")
                 ? operationToDo.opid
                 : null;
             })
           );
         }
+
         setTimeout(function () {
           doOperation(ops.slice(1), theOperation);
         }, operationDelayMs);
       }
     };
 
     if ((fillScriptOps = fillScript.options)) {
-      fillScriptOps.hasOwnProperty("animate") && (animateTheFilling = fillScriptOps.animate),
-        fillScriptOps.hasOwnProperty("markFilling") && (markTheFilling = fillScriptOps.markFilling);
+      // eslint-disable-next-line no-prototype-builtins
+      if (fillScriptOps.hasOwnProperty("animate") && fillScriptOps.animate) {
+        animateTheFilling = fillScriptOps.animate;
+      }
+
+      // eslint-disable-next-line no-prototype-builtins
+      if (fillScriptOps.hasOwnProperty("markFilling") && fillScriptOps.markFilling) {
+        markTheFilling = fillScriptOps.markFilling;
+      }
     }
 
     // don't mark a password filling
     fillScript.itemType && "fillPassword" === fillScript.itemType && (markTheFilling = false);
 
+    // eslint-disable-next-line no-prototype-builtins
     if (!fillScript.hasOwnProperty("script")) {
```

Comment:
```suggestion
    if (!fillScript.script) {
```

Will stop after this one, but yeah same idea.